File: 20_Kontrollstrukturen/1.3.1/Zufallszahl_raten_for.js
import * as random from 'random';
var eingabe, erfolg, x, zahl;
x = random.random();
zahl = Number.parseInt(x * 10) + 1;
erfolg = false;

for (var versuche = 1, _pj_a = 6; versuche < _pj_a; versuche += 1) {
  eingabe = input("Rate meine Zahl zwischen 1 und 10: ");
  versuche += 1;

  if (zahl > Number.parseInt(eingabe)) {
    console.log("Meine Zahl ist gr\u00f6\u00dfer");
  } else {
    if (zahl < Number.parseInt(eingabe)) {
      console.log("Meine Zahl ist kleiner");
    } else {
      console.log("Super. Richtig geraten in %d Versuchen" % versuche);
      erfolg = true;
      break;
    }
  }
}

if (!erfolg) {
  console.log("Du hast das Spiel leider verloren");
}

File: 20_Kontrollstrukturen/1.3.1/Zufallszahl_raten_for.py
import random
x=random.random()
zahl=int(x*10)+1
erfolg=False
for versuche in range(1,6):
    eingabe=input("Rate meine Zahl zwischen 1 und 10: ")
    versuche+=1
    if zahl>int(eingabe):
        print("Meine Zahl ist größer")
    elif zahl<int(eingabe):
        print("Meine Zahl ist kleiner")
    else:
        print("Super. Richtig geraten in %d Versuchen"%(versuche))
        erfolg=True
        break
if not erfolg:
    print("Du hast das Spiel leider verloren")

    
